### ISSUE TITLE
 UIStackView inside UIScrollView?

### ISSUE THREAD

**Issue by JohnReddyZhang** [uchicago-mobi/MPCS51030-2019-Winter-Forum#131]

**Comment by susanstevens** [comment]
Yes, you can certainly put a stack view inside of a scroll view. Remember that there are essentially two ways a stack view's size can be determined: 1) by auto layout constraints between the stack view and its superview, or 2) by the contents of the stack view. 

In your case, you want the contents of the stack view to determine its overall size. I suspect, however, that you may have some constraints on the stack view that are causing the contents to compress, which is why you see the lack of space between the stars and the label.

**Comment by JohnReddyZhang** [comment]
Thank you for the suggestions! I think the text shrink was caused by another constraint I set, which was the height of the image view. I shouldn't have set it to static value.
I'll try to make scrolling work.
Also, could you take a look at the email I just wrote to you discussing about some issues about @w4rner and me? It is kind of urgent so thank you very much! @susanstevens
